k8s_watcher/watcher.go

```diff
@@ -63,13 +65,26 @@ func (c *Watcher) Start() error {
 		return fmt.Errorf("failed to get pod watcher: %v", err)
 	}
 
+	getPodsTicker := time.NewTicker(time.Second * 10)
```

Comment:
it's MUCH more readable when constant is before second like this:
10*time.Second not other way around.